packages/airnode-examples/scripts/make-request.ts

```diff
@@ -8,25 +8,40 @@ import {
   getAirnodeWallet,
   readConfig,
   cliPrint,
+  setMaxPromiseTimeout,
 } from '../src';
 
-const fulfilled = async (requestId: string) => {
+const waitForFulfillment = async (requestId: string) => {
   const airnodeRrp = await getDeployedContract('@api3/airnode-protocol/contracts/rrp/AirnodeRrp.sol');
   const provider = getProvider();
-  return new Promise((resolve) => provider.once(airnodeRrp.filters.FulfilledRequest(null, requestId), resolve));
+
+  const fulfilled = new Promise((resolve) =>
+    provider.once(airnodeRrp.filters.FulfilledRequest(null, requestId), resolve)
+  );
+  const failed = new Promise((resolve) =>
+    provider.once(airnodeRrp.filters.FailedRequest(null, requestId), resolve)
```

Comment:
TODO: I just remembered that we now have an "errorMessage" string on chain so we should show that.